MACILE-guide.md

```diff
@@ -0,0 +1,42 @@
+# MACILE's guide
+
+### Logo
```

Comment:
Remember that headers flow in sequential order. If the title is `h1`, these sections should be `h2`